### DISCUSSION TITLE
Create non-rectangular windows in Tauri applications

### DISCUSSION THREAD

**Discussion by pcgeek86** [tauri-apps/tauri#10076] [General]
Is it possible to create non-rectangular windows in Tauri? I'd like to create a window, but using an entirely custom shape.

**Comment by FabianLars** [comment]
Currently not, no. As far as i know this is also not planned in the near future but feel free to open a feature request (as a normal github issue) for this :)

**Reply by pcgeek86** [reply]
Thanks for the quick response!

**Comment by tyfiero** [comment]
You could make a transparent window and have your UI be transparent as well, then just make the UI look like any shape you want. Not perfect solution, but it would get you close to what you're looking for
